init.lua

```diff
@@ -222,12 +236,12 @@ minetest.register_globalstep(function()
 			local hunger_max = ediblestuff.get_max_hunger(player)
 			local hunger_availabile = hunger_max - ediblestuff.get_hunger(player)
 			local hunger_ratio = hunger_availabile/hunger_max
-			if hunger_ratio >= .15 then -- TODO make this a setting
+			if hunger_ratio >= ediblestuff.hunger_threshold then
```

Comment:
note to self when playtesting: I'll need to test the balance adjustment from 15% to 50% here. It's probably fine.

Reply:
it's fine :)